cl/alerts/templates/disable_docket_alert.html

```diff
@@ -0,0 +1,52 @@
+{% extends "base.html" %}
+{% load text_filters %}
+
+{% block title %}Your Docket Alert is now Disabled – CourtListener.com{% endblock %}
+
+{% block sidebar %}{% endblock %}
+
+{% block content %}
+  <div class="col-sm-2"></div>
+  <div class="col-sm-8">
+    <div class="text-center">
+      <h1>You Have Disabled This Docket Alert</h1>
+      <h3>You're now unsubscribed</h3>
+      <h3 class="gray alt">&mdash; to &mdash;</h3>
```

Comment:
I'm sorry the English language is so hard, but you Subscribe *to* things and Unsubscribe *from* things.